hosts/laptop/configuration.nix

```diff
@@ -14,6 +14,9 @@
     # Use the latest xanmod kernel, mainly for the Clear Linux patches
     boot.kernelPackages = pkgs.linuxPackages_xanmod_latest;
 
+    # TODO: detect from `facter.json`?
```

Comment:
I don't think it honestly makes sense to check like this, because peripherals can be changed at any time. I wouldn't re-generate the `facter.json` just because I bought a new mouse.